proto.sh

```diff
@@ -0,0 +1,11 @@
+echo "Creating proto files"
+./replay_analysis/generated/protoc --proto_path=. --python_out=./replay_analysis/generated ./**/*.proto
+./replay_analysis/generated/protoc --proto_path=. --python_out=./replay_analysis/generated ./api/**/*.proto
```

Comment:
@dtracers Doesn't `**` perform arbitrarily deep matching? Wouldn't `./api/**/*.proto` always be found by `./**/*.proto`?